src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -90,6 +92,7 @@ public function getApplication()
     public function resetApplication()
     {
         codecept_debug('Destroying application');
+        $this->closeSession();
```

Comment:
This completely broke my apps.
I'm not saying this isn't logically correct, but the behaviour changed in a BC-break way.

Reply:
You call this directly from your test code? Does it make sense to mark all client code as internal? Since the whole purpose of the module is to make an API available I'd rather expose it there... The only reason these functions are public is because the module needs to call them.

Reply:
> You call this directly from your test code?

No, I was simply relying on the Session to still be active from one step to another, but destroyed and recreated between different tests.
Again, it may be a bad usage, but was the default one untill this PR, that introduced a breaking change.

Reply:
Hmm, then I assume you have `recreateApplication` enabled.
The docs for that say:
```php
    /**
     * This option is there primarily for backwards compatibility.
     * It means you cannot make any modification to application state inside your app, since they will get discarded.
     * @var bool whether to recreate the whole application before each request
     */
```

In my opinion `recreate the whole application` means recreating the session. 
And the old behavior was therefore a fault.

Note that a breaking changes only apply to `good usage`. If the behavior of the library is not in line with its documentation then that's a bug and if you rely on that behavior you are essentially relying on a abug.

Reply:
> I assume you have `recreateApplication` enabled.

Yup

>  And the old behavior was therefore a fault.
> 
> Note that a breaking changes only apply to `good usage`. If the behavior of the library is not in line with its documentation then that's a bug and if you rely on that behavior you are essentially relying on a abug.

Absolutely right.

Then, I rewrite my first comment: this change completely broke my apps. We should point this bugfix out better in the CHANGELOG.md

Reply:
Yes there's definitely a process gap there.

If you have any good suggestions on a process we're definitely open to it, also feel free to join slack to discuss it there.

/cc @DavertMik

Reply:
How do I find the slack channel of this library?

Reply:
It should be listed here: https://codeception.com/credits 
(but I see it is not).
You can find or slack channel here: https://codeception.slack.com

Reply:
Slack is an invite only chat client (I think). I do not like private converations for FOSS very much: the community is vital for a FOSS to prospere. In other library I've seen gitter in use because it is public (although moderated).